examples/bri-3/src/bri/identity/bpiSubjects/agents/bpiSubjects.agent.ts

```diff
@@ -15,6 +15,6 @@ export class BpiSubjectAgent {
   } 
 
   public createNewExternalBpiSubject(name :string, desc: string, pk: string): BpiSubject {
-    return new BpiSubject(name, desc, BpiSubjectType.External, pk);
+    return new BpiSubject(<BpiSubject>{ name, description: desc, type: BpiSubjectType.External, publicKey: pk });
```

Comment:
nothing big and it is not your change, but since we are modifying this :) lets rename desc to description and pk to publicKey if you dont mind

Reply:
done